config/conf.json

```diff
@@ -1,5 +1,5 @@
 {
-	"webhook": "https://discordapp.com/.......................",
+	"webhook": "DiscordUserHook",
```

Comment:
Why does this need to be changed?

Reply:
When you click on copy URL you get a full string,  I changed it to make swapping it out for the user easier as they just have to paste in contents of the clipboard.